_data/glossary/cronjob.yaml

```diff
@@ -0,0 +1,9 @@
+id: cronjob
+name: CronJob
+tags:
+- core-object
+- workload
+short-description: |
+  Manages time based [Jobs](/docs/concepts/jobs/run-to-completion-finite-workloads/) that are run once or periodically on a given schedule.
```

Comment:
"once" doesn't make sense "on a given schedule".

"A job that runs on a periodic schedule. "?

Reply:
Ok, I'm going with "Manages a [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/) that runs on a periodic schedule."